### ISSUE TITLE
Mapping is not provided between label (Category status) with its respective Check-boxes in multi-select ComboBox.

### ISSUE THREAD

**Issue by SyedRehmanRazvi** [OfficeDev/office-ui-fabric-react#11265]
User Experience:
Users who rely on screen readers are getting affected as screen reader is not announcing label name when focus is on Check-boxes.

Test Environment:
OS: Windows 10
Browser: Microsoft Edge (44.19013.1.0)
URL: https://fabricweb.z5.web.core.windows.net/oufr/7.52.0/#/examples/ComboBox

Repro Steps:
Step 1: Start Narrator 
Step 2: Open the above URL in Edge browser.
Step 3: Navigate to "multi-select ComboBox" and activate it.
Step 4: Verify narrator is announcing label (First Heading) with its respective Check-boxes.

Actual:
Mapping is not provided between label (First Heading) with its respective Check-boxes . So that narrator is not announcing label(First Heading) when focus is on Check-boxes.

Here narrator is announcing as "Option A checkbox, unchecked, 1 of 10"

Expected:
Mapping should be provided between label (First Heading) with its respective Check-boxes. So that narrator will announce label(First Heading) when focus is on Check-boxes.

Here narrator should announce as "First Heading, Option A checkbox, unchecked, 1 of 4"

MAS Reference:
MAS3.3.2
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={4b95a70d-258c-474a-820e-5c8b79d5c0bf}

![image](https://user-images.githubusercontent.com/53210862/69335445-fee62980-0c82-11ea-8c8c-6997611572c9.png)

**Comment by aneeshack4** [comment]
Hi @SyedRehmanRazvi, thank you for reporting this. I see that this repros:
![image](https://user-images.githubusercontent.com/4161791/69389492-099fbf00-0c81-11ea-8158-8aa6b49f10a1.png)

In ComboBox.Basic.Example.tsx, I see that the headings are set the same way as the options with a `key` and `text` object so I'm not sure why Narrator is not picking up on it. The only difference is that this object has an `itemType` of `SelectableOptionMenuItemType.Header` so maybe Narrator for some reason is ignoring text that that itemType. I'll investigate further and follow up.

**Comment by micahgodbolt** [comment]
i'm curious if there are any examples in the wild of this behavior and how they tackled the markup, and the resulting narrator.

**Comment by jspurlin** [comment]
One approach is to have the group (containing div) that contains the header also have an aria-labelledby that points to that header. Not sure if the menu items are structured that way today or if it would need to be reworked to have that structure (I believe it's the latter)

**Comment by aneeshack4** [comment]
Spoke with component owner and looks like this is a dupe of #10921.
